Add March 2023 Newsletter (#1110)
USRSE/usrse.github.io: _posts/newsletters/2023-03-29-newsletter.md

    * [DEI Media Club](#dei-media-club)
    * [Community Calls](#community-calls)
* [Interesting Events and Opportunities](#events)
* [Interesting Reads, Videos, or Podcasts](#reads)
* [US-RSE Outreach](#outreach)
* [US-RSE Financials](#financials)
* [Get Involved](#involved)
* [Recent Job Postings](#jobs)

<a name="conference2023"></a>
# **US-RSE Conference 2023 (US-RSE'23)**

The US-RSE'23 General Chairs will be hosting a [Community Call](#community-calls) on **April 14, 2pm ET/1pm CT/12pm MT/11am PT** for the community to discuss the upcoming conference. We encourage all to join to learn more and ask questions.
    
[Upcoming Submission Deadlines](https://us-rse.org/usrse23/participate/):
    
- **Workshops, Tutorials, and BoF submissions due**: Tuesday, April 4, 2023 - EXTENDED
- **Papers / Notebooks due**: Monday, May 1, 2023
- **Poster / Talk abstracts due**: Monday, June 19, 2023
    
**Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrsecon2023)
    
**Questions?** Contact the organizers at usrsecon2023@easychair.org.

<a name="grant-announcement"></a>
# **Two-year Alfred P. Sloan Foundation Grant**

> We are thrilled to announce that US-RSE has received a two-year grant, totaling nearly $800k, from The Alfred P. Sloan Foundation!

Reviewer: ```suggestion
We are thrilled to announce that US-RSE has received a two-year grant, totaling nearly $800k, from the Alfred P. Sloan Foundation!
```

More information will be available soon, including specific plans, programs, and opportunities to get involved.

We will post updates to the website, Slack, and email over the next few weeks. Additionally, we will host a special [Community Call](https://princeton.zoom.us/meeting/register/tJUud-qvrjMrHdz4NeQEdStLdIhMCPyT1ihr) on **Thursday, April 20 at 2ET/1CT/12MT/11PT** to outline upcoming plans as well as provide an opportunity to answer any questions you may have. This call will be recorded and shared internally on Slack.

Thank you all for your ongoing support of, and participation in, US-RSE. Together, we can continue to make a difference for Research Software Engineers in the US.

<a name="sc-update"></a>
# **Steering Committee Recap**

So far in 2023, the Steering Committee has determined roles for 2023.  See the complete [list on the website](https://us-rse.org/about/steering-committee/). Ian Cosden remains the US-RSE Steering Committee Chair.  
    

> The Steering Committee has also been discussing the move to our new fiscal sponsor, [Community Initiatives](https://us-rse.org/2023-03-21-announcing-ci/), and continuing planning for the 2023 US-RSE Conference. Conference planning has included discussion of potential costs and financial liabilities, such as hotel room blocks. Other topics have included how to keep track of significant organizational developments to track our history, managing permissions for organizational resources, fundraising and US-RSE merchandise, and how to handle requests from external groups for time on [community calls](https://us-rse.org/events/#community-call).

Reviewer: ```suggestion
The Steering Committee also planned and executed the move to our new fiscal sponsor, [Community Initiatives](https://us-rse.org/2023-03-21-announcing-ci/), and continuing planning for the 2023 US-RSE Conference. Conference planning has included discussion of potential costs and financial liabilities, such as hotel room blocks. Other topics have included how to keep track of significant organizational developments to track our history, managing permissions for organizational resources, fundraising and US-RSE merchandise, and how to handle requests from external groups for time on [community calls](https://us-rse.org/events/#community-call).
```

> The Steering Committee also planned and executed the move to our new fiscal sponsor, [Community Initiatives](https://us-rse.org/2023-03-21-announcing-ci/), and continuing planning for the 2023 US-RSE Conference. Conference planning has included discussion of potential costs and financial liabilities, such as hotel room blocks. Other topics have included how to keep track of significant organizational developments to track our history, managing permissions for organizational resources, fundraising and US-RSE merchandise, and how to handle requests from external groups for time on [community calls](https://us-rse.org/events/#community-call).

Reviewer: ```suggestion
The Steering Committee also planned and executed the move to our new fiscal sponsor, [Community Initiatives](https://us-rse.org/2023-03-21-announcing-ci/), and is continuing planning for the 2023 US-RSE Conference. This has included discussion of potential costs and financial liabilities, such as hotel room blocks. Other Steering Committee discussion topics have included how to keep track of significant organizational developments to track our history, managing permissions for organizational resources, fundraising and US-RSE merchandise, and how to handle requests from external groups for time on [community calls](https://us-rse.org/events/#community-call).
```

* [Research Software Engineers: Career Entry Points and Training Gaps](https://doi.org/10.1109/MCSE.2023.3258630) by Ian A. Cosden, Kenton McHenry, and Daniel S. Katz
* [Research Software Engineering at Oak Ridge National Laboratory](https://doi.org/10.1109/MCSE.2023.3260211) by Addi Malviya-Thakur, David E. Bernholdt, William F. Godoy, Gregory R. Watson, Mathieu Doucet, Mark A. Coletti, David M. Rogers, Marshall McDonnell, Jay Jay Billings, and Barney Maccabe
* [Research Software Science: Expanding the Impact of Research Software Engineering](https://doi.org/10.1109/MCSE.2023.3260475) by Michael A. Heroux
    
<a name="funder-series"></a>
## US-RSE Funders Talk Series: National Institutes of Health (NIH) and future talks

US-RSE is happy to be continuing its series of conversations with funders for 1) RSEs to learn about funding opportunities and review and other processes, 2) RSEs to ask questions of the funder, and 3) funders to ask questions and learn about RSEs.

Please join us in this interactive conversation. (If you can’t participate in the interactive event, where you will be able to ask questions and help funders understand US-RSE, it will also be recorded.)

> **Date/Time: [Monday, April 3rd at 1 PM ET, 12 PM CT, 11 AM MT, 10 AM PT](https://us-rse.org/events/2023/2023-04-funder-talk-series/)**

Reviewer 2: ```suggestion
**Date/Time: [Monday, April 3rd at 1pm ET, 12pm CT, 11am MT, 10am PT](https://us-rse.org/events/2023/2023-04-funder-talk-series/)**
```
matches most below and easier to read

**[Register Now!](https://boisestate.zoom.us/meeting/register/tJ0pde2tpjstG9z39q8WfMsTcf3s5GDsbyZ9)**

The NIH Panel includes:
    
* Susan Gregurick, Associate Director for Data Science
* Ishwar Chandramouliswaran, Program Director, Office of Data Science Strategy (ODSS)
* Lori Scott-Sheldon, Program Chief, Data Science and Emerging Methodologies in HIV, Division of AIDS Research, National Institute of Mental Health
* Heidi Sofia, Program Director, National Human Genomic Research Institute (NHGRI)

Following this event with NIH will be talks from:

> * Dr. Carly Strasser, Program Manager for Open Science at the Chan Zuckerberg Initiative (CZI), Tuesday, May 9th at 4 PM ET, 3 PM CT, 2 PM MT, 1 PM PT ([register now](https://us-rse.org/events/2023/2023-05-funder-talk-series/))
> * Dr. Varun Chandola, Program Director in the Office of Advanced Cyberinfrastructure (OAC) at the National Science Foundation (NSF), Tuesday, June 20th at noon ET, 11 AM CT, 10 AM MT, 9 AM PT (registration coming soon)

Reviewer 2: ```suggestion
* Dr. Carly Strasser, Program Manager for Open Science at the Chan Zuckerberg Initiative (CZI), Tuesday, May 9th at 4pm ET, 3pm CT, 2pm MT, 1pm PT ([register now](https://us-rse.org/events/2023/2023-05-funder-talk-series/))
* Dr. Varun Chandola, Program Director in the Office of Advanced Cyberinfrastructure (OAC) at the National Science Foundation (NSF), Tuesday, June 20th at 12pm ET, 11am CT, 10am MT, 9am PT (registration coming soon)
```

<a name="rse-gln"></a>
## RSE Group Leaders' Network (RSE-GLN)   
    

> The RSE-GLN is a group of individuals leading RSE teams who meet to support their peers with the challenges of leading RSE groups.

Reviewer: ```suggestion
The RSE-GLN is a group of individuals leading RSE teams who come together to support their peers in meeting the challenges of leading RSE groups.
```

> The primary membership is for those who currently lead a team and have, or are actively in the process of hiring, at least one direct report; additional members may be invited, such as those who have been in leadership positions in the past. This is to ensure that members have a network of peers in similar situations who are facing similar challenges.  The group meets monthly.

Reviewer: ```suggestion
This group is primarily for those who currently lead a team and have, or are actively in the process of hiring, at least one direct report. Additional members may also be invited, such as those who have been in leadership positions in the past. This is to ensure that members have a network of peers in similar situations who are facing similar challenges.  The group meets monthly.
```

The RSE-GLN also seeks to support aspiring RSE Group Leaders through a separate mentorship and development network. The Aspiring RSE-GLN is open to those looking to build RSE teams at their institution or move into RSE group leadership roles. There are no formal requirements, but those with the experience and qualifications to take on leadership roles will likely benefit more than early career RSEs.
    
[Learn more and register.](https://us-rse.org/ag/rse-gln)

<a name="dei-speaker-series"></a>
## US-RSE DEI-WG Speaker Series

> The US-RSE Diversity, Equity, and Inclusion Working Group (DEI-WG) is working to bring to the US-RSE community quarterly visits from RSE professionals with vested interests or research into DEI in the RSE community at large. Our community pledges to continually educate ourselves through research on best practices focused on diversity, equity, and inclusion. The US-RSE is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community. We encourage everyone to suggest speakers and topics and are open to different formats and timelines of the events. Feedback is much appreciated.

Reviewer: ```suggestion
The US-RSE Diversity, Equity, and Inclusion Working Group (DEI-WG) is working provide the US-RSE community quarterly visits from RSE professionals with vested interests or research into DEI in the RSE community at large. Our community pledges to continually educate ourselves through research on best practices focused on diversity, equity, and inclusion. The US-RSE is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community. We encourage everyone to suggest speakers and topics and are open to different formats and timelines of the events. Feedback is much appreciated.
```

Reviewer 2: (you mean "to provide"?)

<a name="dei-media-club"></a>
## US-RSE DEI-WG Media Club

The DEI Working Group hosts a regular Media Club discussion in which we invite participants to read, listen to, or watch the chosen DEI media and come ready to discuss the content as a small group. Next up is a discussion of this podcast on the Florida’s Department of Education rejecting an AP course on African American Studies, and what's happening in higher education there in general. [Listen to the podcast](https://voltedu.com/podcast/podcast-crossover-the-college-boards-trouble-in-florida/).

[Link to register](https://stanford.zoom.us/meeting/register/tJwkce-uqjwoHdxLgipTO2ncD943cXWpmAeJ)

<a name="community-calls"></a>
## Community Calls
    
### April Community Call
    
The April community call will be on Friday, **April 14, 2pm ET/1pm CT/12pm MT/11am PT**.

> The topic of the community call will be the first ever US-RSE conference, which will take place in Chicago, IL, October 16-18, 2023. We are committed to making this event successful! To ensure its usefulness to the US-RSE community, we’d like to hear your thoughts and ideas! That’s why the next community call will be all about the conference. Are there specific events you would like to see? What would or wouldn’t make you attend? Do you have concrete ideas of how this conference could be useful to you? Join us and discuss with other members of the community.

Reviewer: ```suggestion
The topic of the community call will be the first ever US-RSE conference, which will take place in Chicago, IL, October 16-18, 2023. We are committed to making this event successful! To ensure its usefulness to the US-RSE community, we’d like to hear your thoughts and ideas! That’s why this community call will be all about the conference. Are there specific events you would like to see? What would or wouldn’t make you attend? Do you have concrete ideas on how this conference could be useful to you? Join us and discuss with other members of the community.
```

    
**Registration details**
You can register for this community call [here](https://asu.zoom.us/meeting/register/tZUkdOqrrDgqHdKXUbh__bHT-ENeSWMrdgLK).
    
### Community Call - Special Edition
    

> On Thursday, **April 20 at 2pm ET/1pm CT/12pm MT/11am PT** we will host a special Community Call to outline upcoming plans for [our new grant](https://us-rse.org/2023-03-27-grant-announcement/) that US-RSE has received from The Alfred P. Sloan Foundation. This will provide an opportunity to answer any questions you may have. The call will be recorded and shared internally on Slack.

Reviewer: ```suggestion
On Thursday, **April 20 at 2pm ET/1pm CT/12pm MT/11am PT** we will host a special Community Call to outline upcoming plans for [the new grant](https://us-rse.org/2023-03-27-grant-announcement/) that US-RSE has received from the Alfred P. Sloan Foundation. This will provide an opportunity to answer any questions you may have. The call will be recorded and shared internally on Slack.
```

**Registration details**
You can register for this special Community Call [here](https://princeton.zoom.us/meeting/register/tJUud-qvrjMrHdz4NeQEdStLdIhMCPyT1ihr).

<a name="events"></a>
# **Interesting Events and Opportunities**

## INTERSECT RSEng Bootcamp - July 10-14, 2023

> This 4.5 day event, targeting those who self-identify as intermediate research software developers with backgrounds in a research domain other than computer science, will cover RSE topics including:

Reviewer: ```suggestion
This 4.5-day event, targeting those who self-identify as intermediate research software developers with backgrounds in a research domain other than computer science, will cover RSE topics including:
```

* Software Design
* Working Collaboratively
* Testing
* CI/CD
* Packaging & Distribution

> Travel funding is available! For more information or to apply, visit https://intersect-training.org/bootcamp23/.

Reviewer: ```suggestion
Travel funding is available! For more information or to apply, visit [https://intersect-training.org/bootcamp23/](https://intersect-training.org/bootcamp23/).
```

## Upcoming Events
    
* [Research Software Engineers in HPC Minisymposium](https://pasc23.pasc-conference.org/program/minisymposia/) - as part of PASC23 in Davos, Switzerland June 26-28
* Software Engineering for Research Software ([SE4RS’23](https://se4science.org/workshops/se4rs23/index.htm)) - to be held in conjunction with PEARC’23 in Portland, OR July 23-27
* Software Engineering for eScience ([SE4eScience’23](https://se4science.org/workshops/se4escience23/)) - to be held in conjunction with eScience’23 in Cyprus October 9-13
* Research Software Engineers in eScience ([RSE-eScience-2023](https://us-rse.org/rse-escience-2023/)) - to be held in conjunction with eScience’23 in Cyprus October 9-13
* Research Software Engineers in HPC (RSE-HPC-2023) - to be held in conjunction with SC23 in Denver, CO, November 12-17

<a name="reads"></a>
# **Interesting Reads, Videos, or Podcasts**

A note from Peter Schmidt on the two year anniversary of his podcast, [Code for Thought](https://codeforthought.buzzsprout.com):

<blockquote>Two Years of Code for Thought
<br>
<br>

> On 12 January 2021, the first episode of Code for Thought, the podcast show for research and software engineering, went live with [‘And so it begins…’](https://codeforthought.buzzsprout.com/1326658/7121884-and-so-it-begins) At the time, I was quite nervous about launching the podcast. After all, I just barely started co-hosting podcast episodes with the wonderful Vanessa Sochat and her ‘RSE Stories’ show (now [Developer Stories](https://rseng.github.io/devstories/). On top of that, with podcasting being very popular and so many shows to choose from, where would this one end up?

Reviewer: ```suggestion
On 12 January 2021, the first episode of Code for Thought, the podcast show for research and software engineering, went live with <a href="https://codeforthought.buzzsprout.com/1326658/7121884-and-so-it-begins">‘And so it begins…’</a> At the time, I was quite nervous about launching the podcast. After all, I just barely started co-hosting podcast episodes with the wonderful Vanessa Sochat and her ‘RSE Stories’ show (now <a href="https://rseng.github.io/devstories/">Developer Stories</a>. On top of that, with podcasting being very popular and so many shows to choose from, where would this one end up?
```

> On 12 January 2021, the first episode of Code for Thought, the podcast show for research and software engineering, went live with <a href="https://codeforthought.buzzsprout.com/1326658/7121884-and-so-it-begins">‘And so it begins…’</a> At the time, I was quite nervous about launching the podcast. After all, I just barely started co-hosting podcast episodes with the wonderful Vanessa Sochat and her ‘RSE Stories’ show (now <a href="https://rseng.github.io/devstories/">Developer Stories</a>. On top of that, with podcasting being very popular and so many shows to choose from, where would this one end up?

Reviewer: ```suggestion
On 12 January 2021, the first episode of Code for Thought, the podcast show for research and software engineering, went live with <a href="https://codeforthought.buzzsprout.com/1326658/7121884-and-so-it-begins">‘And so it begins…’</a> At the time, I was quite nervous about launching the podcast. After all, I just barely started co-hosting podcast episodes with the wonderful Vanessa Sochat and her ‘RSE Stories’ show (now <a href="https://rseng.github.io/devstories/">Developer Stories</a>). On top of that, with podcasting being very popular and so many shows to choose from, where would this one end up?
```

<br>
<br>
At the same time, the questions that have puzzled us RSEs for so long still persist: How do we get recognition and proper support for the work we do? And while in some countries, like the UK, research centres and universities have been building teams of RSEs, in other countries such roles still need to be defined and funded.
<br>
<br>
All of this means that the work of raising awareness of research software engineering and growing our community must continue. And it should do so on an international level as well as on a local and national one.
<br>
<br>
Podcasting is a great way of helping to achieve that and I am delighted to continue and even increase the effort on Code for Thought. This means increasing the visibility of research and software engineering not only in the UK but elsewhere in the world.
<br>
<br>
In the annual report from the Code for Thought hosting site I read that the podcast has been listened to by people from 59 different countries. Which blew my mind away. But it also showed that the podcast needs to serve a wider community.
<br>
<br>

> To keep this podcast as a true community effort it is important to hear from you! I look forward to hear about your ideas, comments and suggestions on <a href="mailto:code4thought@proton.me">email</a>, <a href="https://www.linkedin.com/company/codeforthought/">LinkedIn</a>, <a href="https://ukrse.slack.com/join/signup#/domain-signup">UK Slack workspace</a> (@code4thought), <a href="https://fosstodon.org/@code4thought">Mastodon</a>, and <a href="https://code4thought-podcast.github.io/newsletters/">Github news/blogs</a>. 

Reviewer: ```suggestion
To keep this podcast as a true community effort it is important to hear from you! I look forward to hear about your ideas, comments and suggestions on <a href="mailto:code4thought@proton.me">email</a>, <a href="https://www.linkedin.com/company/codeforthought/">LinkedIn</a>, <a href="https://ukrse.slack.com/join/signup#/domain-signup">UK Slack workspace</a> (@code4thought), <a href="https://fosstodon.org/@code4thought">Mastodon</a>, and <a href="https://code4thought-podcast.github.io/newsletters/">GitHub news/blogs</a>. 
```

<br>
<br>
My heartfelt thanks for listening and supporting this podcast. I wish you all the best for 2023 and I hope you enjoy the show.
<br>
<br>
Peter
</blockquote>

Items you may have missed on the blog and Slack:
    
  * Blogs and podcasts with US-RSE members
     * [Defining the roles of research software](https://doi.org/10.54900/9akm9y5-5ject5y) by Rob van Nieuwpoort and Daniel S. Katz
     * [Developer Stories Podcast](https://rseng.github.io/devstories/posts/)
  * Blogs and podcasts from beyond US-RSE
     * [Code for Thought Podcasts](https://codeforthought.buzzsprout.com)
     * [Sustaining Open Source Ecosystems Podcasts](https://podcast.sustainoss.org)

* The organizers of RSECon2022 have released a playlist with 53 videos so far 
    * <a name="https://www.youtube.com/playlist?list=PL27mQJy8eDHk---gHCapHwyVaWIkyORYj"></a>[RSECon2022 Playlist](https://www.youtube.com/playlist?list=PL27mQJy8eDHk---gHCapHwyVaWIkyORYj)

* The latest [Research Software Alliance (ReSA) newsletter](https://preview.mailerlite.com/p3p4u2g7h9/2175082983801755815/u9c5/) includes a lot of research software community news and events.

<a name="outreach"></a>
# **US-RSE Outreach**

> The Outreach Working Group is looking for volunteers to help with organizing workshops, birds of a feather (BoF), symposiums, etc. Roles span from organizer to reviewer to speaker and more.

Reviewer: ```suggestion
The Outreach Working Group is looking for volunteers to help with organizing workshops, birds of a feather (BoF), symposiums, etc. Roles range from organizer to reviewer to speaker and more.
```

> The Outreach Working Group is looking for volunteers to help with organizing workshops, birds of a feather (BoF), symposiums, etc. Roles range from organizer to reviewer to speaker and more.

Reviewer: ```suggestion
The Outreach Working Group is looking for volunteers to help with organizing workshops, birds of a feather (BoFs), symposiums, etc. Roles range from organizer to reviewer to speaker and more.
```

If you are interested in getting involved in a US-RSE hosted event, please reach out to the working group via the [#wg-outreach](https://app.slack.com/client/T8ZT4PJSW/CNR6E0N75) slack channel or [wg-outreach@us-rse.org](mailto:wg-outreach@us-rse.org).

<a name="financials"></a>
# **US-RSE Has a New Fiscal Sponsor!**

> We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally sponsored project of [Community Initiatives](http://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.

Reviewer: ```suggestion
We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally-sponsored project of [Community Initiatives](http://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.
```

Reviewer 2: I'd disagree with this one given the `-ly` suffix of the adverb.

Reviewer: to me, "fiscally-sponsored" is an adjective phrase that should be hyphenated. It's a question of style, however.

> We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally-sponsored project of [Community Initiatives](http://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.

Reviewer 2: ```suggestion
We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally sponsored project of [Community Initiatives](http://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.
```
but happy to agree to differ if you prefer the hyphen

Author: You and Dan Katz over here, arguing about hyphens... XD

Reviewer: I don't care that much

Author: I think I'll remove it only so it matches the blog post that was made about this change.

<a name="involved"></a>
# **Get Involved**

> There are lots of ways to get involved with the US-RSE community. Of course, you can [join us on Slack](https://us-rse.org/join). We’re also looking for ideas and help in many other places. See our [list of projects](https://docs.google.com/document/d/1jjVD0WkeeWZJI6yqSKyMdIjtClzolsxv75RkpLju17I/edit?usp=sharing) and let us know how you’d like to help. Help with things like the Website Committee, Social Media, Community Engagement, Fundraising, and Newsletter are all needed and welcome.

Reviewer: ```suggestion
There are lots of ways to get involved with the US-RSE community. Of course, you can [join us on Slack](https://us-rse.org/join). We’re also looking for ideas and help in many other places. See our [list of projects](https://docs.google.com/document/d/1jjVD0WkeeWZJI6yqSKyMdIjtClzolsxv75RkpLju17I/edit?usp=sharing) and let us know how you’d like to help. Help with things like the Website Committee, Social Media, Community Engagement, Fundraising, and Newsletter is needed and welcome.
```
